bootcamp-java/Topic1/classes/Offers.java

```diff
@@ -0,0 +1,40 @@
+package classes;
+
+public class Offers {

```

Comment:
ok, lo modelo de esa manera. Buenísimo

Date: Tue, 20 Jan 2015 07:00:03 -0800
From: notifications@github.com
To: java-bootcamp@noreply.github.com
CC: marcosadriandalmasso@hotmail.com
Subject: Re: [java-bootcamp] ShoppingCart (#4)

In bootcamp-java/Topic1/classes/Offers.java:

> @@ -0,0 +1,40 @@
> +package classes;
> +
> +public class Offers {

Por ahi hacer que oferta herede de producto no sería muy claro. Yo lo pensaría como una relación (una composición). Por ejemplo, oferta tiene una relación al producto y un campo con el nuevo precio o con el % de descuento.


—
Reply to this email directly or view it on GitHub.

  
    
    
  
  
 		 	   		  =